coverage-tests.js

```diff
@@ -1604,6 +1604,23 @@ test('check window fully with html scrollRootElement after scroll', {
   }
 })
 
+test('check window fully with html scrollRootElement after scroll when fail to scroll', {
+  variants: {
+    'with css stitching': {config: {stitchMode: 'CSS', baselineName: 'TestCheckWindowFullyWithHtmlScrollRootElementAfterScroll'}},
+    'with scroll stitching': {config: {stitchMode: 'Scroll', baselineName: 'TestCheckWindowFullyWithHtmlScrollRootElementAfterScroll_Scroll'}},
+  },
+  test({driver, eyes}) {
+    driver.visit('https://demo.applitools.com')
```

Comment:
I think a better practice will be to add this page to the list of named pages, with a proper name (what is special about this page?).